### PR TITLE
create github-mgmt stewards team

### PR DESCRIPTION
**Pull request by galargh**
This PR creates a new team called `github-mgmt stewards` and grants it write access to `github-mgmt` repository. 

The general idea is to build up `github-mgmt stewards` team with users who are:
- familiar with GitHub Management
- trusted within the org
- want/need to be able to triage/merge PRs

`github-mgmt stewards` should be the only team with write access to `github-mgmt` repository, eventually

Here's my proposal for initial members:
- @aschmahmann
- @BigLep
- @marten-seemann
- @MarcoPolo
- @mxinden
- @p-shahi
- @willscott

Here is a short intro doc to GitHub Management Stewards I prepared - https://github.com/libp2p/github-mgmt/blob/e0283a36e238ce60110fb86a80eefcbbb621ebdf/docs/STEWARDSHIP.md This is intended only as a v0 and should definitely be updated according to how the team in this org intends to organise their work.

Similar change was introduced recently in the `ipfs` org. See https://github.com/ipfs/github-mgmt/pull/37 for more details.

###### TODO
- [x] get approval for initial github-mgmt stewards team members from Steve
- [x] prepare 1 page document on what is expected of a github-mgmt steward 
- [x] mention proposed members once the PR is ready for review

### CODE REVIEW

File: github/libp2p.yml [hunk 1]
```diff
@@ -3113,6 +3122,25 @@ teams:
...
+    #  using a team instead of direct collaborators because we want to reference it in the CODEOWNERS file
+    create_default_maintainer: false
+    description: Users that are effectively org admins
+    members:
+      # WARN: membership here should be treated exactly as cautiosly as having an org admin role
+      # ATTN: members are expected to:
+      #  - be familiar with GitHub Management
+      #  - be ready to triage/review org configuration change request in github-mgmt
+      maintainer:
+        - aschmahmann
```
Comment (BigLep) [inline github/libp2p.yml:3136]: Make a member

File: github/libp2p.yml [hunk 2]
```diff
@@ -3113,6 +3122,25 @@ teams:
...
+        - BigLep
+        - marten-seemann
+      member:
+        - MarcoPolo
+        - mxinden
```
Comment (BigLep) [inline github/libp2p.yml:3141]: Make a maintainer

Comment (BigLep) [reply github/libp2p.yml:3141]: Add p-shahi

File: github/libp2p.yml (hunk 1 above)
Comment (galargh) [reply github/libp2p.yml:3136]: Adin cannot be "just" a member because of his admin role in the organization.

File: docs/STEWARDSHIP.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+- review and merge changes to the GitHub Management configuration
+
+### How to review and merge a GitHub Management pull request?
+
+- Wait for the `Comment` check to pass
+- Verify that the pull request contains only the changes you expect
+- Verify that the plan posted as a comment introduces **only** the changes you expect
+- Check if there are any open PRs created by the `Sync` workflow (titles starting with `sync`) and merge them first if there are
+- Ask the author of the pull request to provide more context if needed
+- Do **not** update the pull request yourself - let the `Update` workflow do it
```
Comment (galargh) [inline docs/STEWARDSHIP.md:26]: This is no longer required, see https://github.com/ipld/github-mgmt/pull/21#discussion_r970641541 for details.
```suggestion
```

**Review by BigLep — APPROVED** [review]

**Pull request merged**